build.py

```diff
@@ -80,6 +80,7 @@ def __env(envdir):
 def env():
     """set up a base virtual environment"""
     _env()
+    _deps()
```

Comment:
No, now that we use tox the env's basic task is only to bootstrap tox if necessary.